korangar/src/loaders/model/mod.rs

```diff
@@ -374,13 +430,22 @@ impl ModelLoader {
             })
             .collect();
 
-        let root_node_name = &model_data.root_node_name.clone().unwrap();
+        let mut root_node_names = Vec::<ModelString<40>>::new();
 
+        if version.equals_or_above(2, 2) {
+            for i in 0..model_data.root_node_count.unwrap() {
+                root_node_names.push(model_data.root_node_names[i as usize].clone());
+            }
+        } else {
+            let root_node_name = model_data.root_node_name.clone().unwrap();
+            root_node_names.push(root_node_name);
+        }
+        // TODO: CHECK FOR OTHER ROOT NODES
```

Comment:
Do you want to do this in this PR? Our model structure should allow for multiple root nodes already.

Reply:
I was thinking of it, but doesn't find any model with multiple root nodes. So I postponed.